README.md

```diff
@@ -684,7 +721,82 @@ usr/src/uts/intel/os/name_to_major
 usr/src/uts/intel/os/minor_perm
 ```
 
-## Testing and Things to Think About
+# Contributing
+
+All the repositories contained within this build use github pull requests for
+new changes.
+
+All changes should have an associated issue. You can use the [GitHub
+issue tracker](https://github.com/joyent/smartos-live/issues). Joyent
+employees use an internal JIRA exposed at
+<https://smartos.org/bugview>. The commit message should be of the form used by
+illumos-gate, for example:
+
+```
+smartos-live#9999 make some changes
```

Comment:
I don't have a preference at all, but my understanding was that we tended to include the "joyent/" prefix on the issue name. E.g.:

```
commit 0158021c21657bdc537895a634d1a73b593e6639
Author: Todd Whiteman <todd.whiteman@joyent.com>
Date:   2019-05-03T18:29:31-07:00 (6 months ago)

    joyent/smartos-live#771 imgadm vacuum will try to reap images used by bhyve vms
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>
```

Me testing GitHub automatic linking:
- without joyent prefix: smartos-live#771
- with joyent prefix: joyent/smartos-live#771

Reply:
Yah, the "with prefix" form links up automatically, FWIW.

Reply:
Done, also updated this for the newer RFD175 format.